frontend/src/store/actions/index.js

```diff
@@ -0,0 +1,117 @@
+import axios from 'axios';
+//Article
+export const ADD_ARTICLE = 'ADD_ARTICLE';
+export const DELETE_ARTICLE = 'DELETE_ARTICLE';
+export const FETCH_ARTICLE_DATA = 'FETCH_ARTICLE_DATA';
+export const ARTICLE_DATA_FETCHED = 'ARTICLE_DATA_FETCHED';
+export const ERROR = 'ERROR';
```

Comment:
To separate functionality, and possibly prevent weird issues later when the error for articles needs to do something different from the error in users, I'd suggest a separate ARTICLE_ERROR and USER_ERROR.